### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by ernmander** [elementary/calendar#239]
reminders / notifications don't seem to work with the pre installed calendar.

**Comment by pinkcichlid** [comment]
have same issue reminders do nothing

**Comment by laubster** [comment]
ditto

**Comment by CodingSpiderFox** [comment]
Please describe in detail: what did you expect and what happened instead

I had an issue today: I set an appointment for 9:38 AM with notification 10 minutes in advance. It was already actually 9:37 AM when I was doing this. So I would expect that the notification appears straight away. But instead, at 11:29 AM, I got a notification that said "Test (in 9 minutes)"

**Comment by bbuhler** [comment]
For me the notification comes almost 2 hours after the event happend:

![Screenshot from 2019-09-19 15-53-51](https://user-images.githubusercontent.com/2802675/65250678-3a833c80-daf6-11e9-906c-b6b1c1912f70.png)

I assume it is a timezone bug.

**Comment by Juno37** [comment]
Reminder does not appear in notification pop up.

Reminder lot listed in missed notifications list.

:-(

Juno 5.0

**Comment by jtrees** [comment]
I think this may be related to timezone issues (#92). I just got a notification 1 hour late and have my timezone set to UTC+1. Could this be the case for you too, @bbuhler?

**Comment by camilova** [comment]
For me the error is due timezones, because I use GMT-4 and when I set an event at 17:00 h localzone (GMT-4) the alert shows at 13:00 h, als it were UTC. (#92)

**Comment by mcclurgm** [comment]
Good points, but I don't think that's everything. I'm working on a time zone fix, and notifications haven't improved for me. They deliver at weird times, and often not at all. So while timezones may fix issues like @jtrees and @camilova are having, there's definitely another issue going on here.

**Comment by fabiozaramella** [comment]
Whilst time zones still don't seem to be recognized correctly, notifications are working now after #637 also this is a duplicate of both #474 and #92 so i'm going to close it.
